spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/logback/LogbackConfigurator.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2012-2023 the original author or authors.
+ * Copyright 2012-2024 the original author or authors.
```

Comment:
There doesn't seem to be any changes in this file so the copyright header should not be updated.

Reply:
This file was updated in #41885, which was reverted, but updated again without updating the years. So I feel it's accurate, but I'm happy to take it out if required.

Reply:
I should have bumped the year as part of https://github.com/spring-projects/spring-boot/commit/dd2ed5f12d23b15ff0055e366ab6da8e95218596. That should now be done separately as it isn't related to this change so please take it out of this PR.